cli/tests/integration/coverage_tests.rs

```diff
@@ -90,20 +92,18 @@ fn run_coverage_text(test_name: &str, extension: &str) {
   let tempdir = TempDir::new();
```

Comment:
We could incrementally improve this in a future PR, but I think most `TempDir::new()` calls could be removed and instead do:

```rs
let context = TestContext::default();
let temp_dir = context.deno_dir();
```

In most cases it's ok to re-use the temp deno dir that's created (or use a subfolder of it) in order to avoid bothering to create additional temp directories.

Reply:
I've updated the tempdirs to deno_dirs, still need to run it in the CI but locally it seems to be working fine